app/services/account_validation_service.rb

```diff
@@ -0,0 +1,10 @@
+class AccountValidationService
```

Comment:
nit: RuboCop extension has some formatting suggestions

Reply:
Thanks for flagging, I fixed the errors